packages/vm/lib/state/interface.ts

```diff
@@ -1,4 +1,4 @@
-import Account from '@ethereumjs/account'
+import { Account } from 'ethereumjs-util'
```

Comment:
Ah ok, so this DOES change the `StateManager` interface, also something worth to be mentioned in the release notes.